docs/developers/components/README.md

```diff
@@ -1,34 +1,268 @@
 # Components
 
-HarperDB is a highly extensible database application platform with support for a rich variety of composable modular components and components that can be used and combined to build applications and add functionality to existing applications. HarperDB tools, components, and add-ons can be found in a few places:
+Harper is not just a database, but also a highly extensible, JavaScript application platform built on a concept called **components**. Harper components are many things, and at the highest level are defined as JavaScript based _extensions_ of the core Harper platform. They are executed by Harper directly; thus, have complete access to the Harper Global APIs (such as `Resource`, `databases`, and `tables`).
```

Comment:
IMO if we could get away from leading with or saying "not just a database" the message & branding will be less confusing & sounds stronger.

Perhaps replace with:
Harper components are a core Harper concept defined as flexible JavaScript based _extensions_ of the highly extensible core Harper platform. They are executed by Harper directly; thus, have complete access to the Harper Global APIs (such as `Resource`, `databases`, and `tables`).